umm-spec-lib/src/cmr/umm_spec/dif_util.clj

```diff
@@ -106,7 +106,7 @@
  "Map a combination of UMM URLContentType, Type, and Subtype (optional) to a dif url content type
  type and subtype. This is not dif->umm list flipped."
  {{:URLContentType "CollectionURL" :Type "DATA SET LANDING PAGE"} ["VIEW DATA SET LANDING PAGE" nil]
-  {:URLContentType "CollectionURL" :Type "DOI"} ["VIEW DATA SET LANDING PAGE" nil]
+  {:URLContentType "CollectionURL" :Type "DOI"} ["Erich needs to change this. VIEW DATA SET LANDING PAGE" nil]
```

Comment:
@eereiter, please see my comments above. we can't have both "DOI" and "DATA SET LANDING PAGE" map to the same thing - that's how I understand it at least.

Reply:
Yes we can, otherwise we would be dropping the URL data.